src/pages/MetricDetail.svelte

```diff
@@ -6,12 +6,15 @@
   import HelpHoverable from "../components/HelpHoverable.svelte";
   import helpText from "../data/help";
   import PageTitle from "../components/PageTitle.svelte";
+  import { pageTitle } from "../state/stores";
 
   export let params;
   let metricName = params.metric.replaceAll("-", ".");
 
   const metricDataPromise = getMetricData(params.app, metricName);
 
+  pageTitle.set(`${metricName} | Metric `);
```

Comment:
Let's include the app name -- I think *that* something is a metric is usually fairly obvious from the title.

```suggestion
  pageTitle.set(`${metricName} | ${params.app} `);
```